api/scpca_portal/models/sample.py

```diff
@@ -147,6 +147,11 @@ def get_metadata(self) -> Dict:
 
         return sample_metadata
 
+    def get_lock_name(self, download_config: Dict) -> str:
+        """Returns a unique lock name based on multiplexed status and download config"""
+        lock_prefix = self.scpca_id if not self.is_multiplexed else "_".join(self.multiplexed_ids)
+        return f'{lock_prefix}-{download_config["modality"]}-{download_config["format"]}'
+
```

Comment:
```suggestion
    def get_config_identifier(self, download_config: Dict) -> str:
        """Returns a unique identifier for the sample and download config combination. Multiplexed samples share an ID and are not unique."""
        sample_id = self.scpca_id if not self.is_multiplexed else "_".join(self.multiplexed_ids)
        return "_".join([sample_id] + sorted(download_config.values()))
```

What do you think about this change? I like the idea of moving away from "lock" in the function name as it doesn't really matter, we just want to indicate that the processing is not unique which is why they share a name.

Reply:
Yeah, this is great. I'll make this addition.